### DISCUSSION TITLE
Exposing More Than One Endpoint Or Port In Container

### DISCUSSION THREAD

**Discussion by wisamidris7** [microsoft/aspire#3457] [Q&A] [answered]
When trying to make rabbitmq:3-management

This is what i did

```c#
var rabbitMq = builder.AddRabbitMQContainer("eventbus", port: 9293, password: "secret")
    .WithAnnotation(new ContainerImageAnnotation() {
        Image = "rabbitmq",
        Tag = "3-management"
    })
    .WithAnnotation(new EndpointAnnotation(ProtocolType.Tcp, port: 15672, containerPort: 15672, name: "Management"));
```

But in

```c#
builder.Build().Run();
```

It throws Squence Contains... Expection 
So what if aspire contains exposing more than one port in aspire for container and thanks!

This is image of full excpetion

![image](https://github.com/dotnet/aspire/assets/104096256/5086149d-a09f-48da-8bc1-297a9544142c)

**Comment by davidfowl** [comment, marked as answer]
Upgrade your aspire packages and workload to preview 4.

**Comment by davidfowl** [comment]
BTW the workload is dead, update to aspire 9.x!
